.github/workflows/build-wheel-linux-arm64.yaml

```diff
@@ -163,6 +163,7 @@ jobs:
     needs: [constants, build-dependencies]
     strategy:
       fail-fast: false
+      max-parallel: 2
```

Comment:
Oh is this new?

Reply:
Yes, from RC, otherwise the runner goes into a bad state, because it is building wheels for MacOS and Linux Arm64 at the same time.